src/main/java/com/es/boardGameTraining/model/Game.java

```diff
@@ -0,0 +1,171 @@
+package com.es.boardGameTraining.model;
+
+import jakarta.persistence.*;
+
+import java.util.List;
+
+@Entity
+@Table(name = "games")
+public class Game {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
```

Comment:
Is this `id` the `bggId`?

I think that you should have 2 separates IDs, your database ID and another for the `bggId`

In the future maybe you can add games not related to the BGG, and the `bggId` should be nullable

Reply:
Maybe you have to change the DTO class